_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -28,7 +28,7 @@ The rule applies to any [keyboard event][]:
 For each test target:
```

Comment:
Name the conditions as in #1275 so you'll be able to easily refer to them in the descriptions.